babel.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  presets: ['@babel/preset-env', '@babel/react', '@babel/flow'],
```

Comment:
Should we be configuring `preset-env` to match the browser list in pinboard?

Reply:
@btraut 
> By default @babel/preset-env will use browserslist config sources unless either the targets or ignoreBrowserslistConfig options are set.
https://babeljs.io/docs/en/babel-preset-env#browserslist-integration

So updating it with the settings from pinboard results in the following before / after for browsers:

Before
```
and_chr 78
and_ff 68
and_qq 1.2
and_uc 12.12
android 76
baidu 7.12
bb 10
bb 7
chrome 78
chrome 77
chrome 76
chrome 49
edge 18
edge 17
firefox 70
firefox 69
firefox 68
ie 11
ie 10
ie_mob 11
ie_mob 10
ios_saf 13.2
ios_saf 13.0-13.1
ios_saf 12.2-12.4
ios_saf 8
kaios 2.5
op_mini all
op_mob 46
op_mob 12.1
opera 64
opera 63
safari 13
safari 12.1
safari 5.1
samsung 10.1
samsung 9.2
```

After
```
and_chr 78
and_uc 12.12
chrome 78
chrome 77
edge 18
firefox 69
ie 11
ios_saf 13.0-13.1
op_mini all
safari 13
samsung 10.1
```

For reference, you can test this with `npx browserslist`